internal/handler/msg_aggregate.go

```diff
@@ -401,8 +403,7 @@ func (h *Handler) MsgAggregate(connCtx context.Context, msg *wire.OpMsg) (*wire.
 		cursor.Close()
 	}
 
-	var reply wire.OpMsg
-	must.NoError(reply.SetSections(wire.MakeOpMsgSection(
+	return wire.NewOpMsg(must.NotFail(bson.ConvertDocument(
```

Comment:
Since we added `opMsgDocument` to convert `*wire.OpMsg` to `*types.Document`, let's add a mirrored helper to convert `*types.Document` to `*wire.OpMsg` . Then the last line could look just like
```go
return documentOpMsg(
// ...
```

Reply:
Sure, good idea! Added a helper.